react/features/app/components/App.web.js

```diff
@@ -38,6 +41,31 @@ export class App extends AbstractApp {
     }
 
     /**
+     *  Sets handler for beforeunload event after mounting of the component.
+     *
+     *  @returns {void}
+     */
+    componentDidMount() {
+        // Stops collecting the logs and disposing the API when the user
+        // closes the page.
+        const store = this._getStore();
+        const {
+            logCollector,
+            logCollectorStarted
+        } = store.getState()['features/base/logger'];
+
+        window.addEventListener('beforeunload', () => {
+            // Stop the LogCollector
+            if (logCollectorStarted) {
+                logCollector.stop();
+                store.dispatch(startLogCollector(false));
```

Comment:
I find the function name `startLogCollector` with an argument `isStarted` that either starts or stops the log collector not a good choice. If you don't want to have `startLogCollector()` & `stopLogCollector()`, then please go with `setLogCollectorStarted(boolean)` similar to the common function name `setEnabled(boolean)`. Moreover, you already defined the redux state `logCollectorStarted` so my proposed function name sounds to me to be in line.